src/QueueReceiver.Core/Services/GraphService.cs

```diff
@@ -4,9 +4,11 @@
 using QueueReceiver.Core.Interfaces;
 using QueueReceiver.Core.Models;
 using QueueReceiver.Core.Settings;
+using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net.Http.Headers;
 using System.Threading.Tasks;
-using System;
 
 namespace QueueReceiver.Core.Services
 {
```

Comment:
Method GetPersonByOid swallow the exception. Should it?

Reply:
good question. We have a mismatch of interest here. We wan't it for the sync service, but not necessarily for QueueReceiverService.

Reply:
I've moved the try catch to syncservice.